src/app/modules/shared_models/token_list_item.nim

```diff
@@ -14,6 +14,8 @@ type
     image*: string
     category*: int
     communityId*: string
+    supply*: string
```

Comment:
Should this really be a `string`?

Reply:
Imo string is ok here, especially if we want to keep supply (not sure if now) expressed in base units where int or float may be insufficient.